src/pages/EventDetailsPage/components/CheckOffTable/index.tsx

```diff
@@ -57,28 +56,28 @@ function CheckOffTable(props: CheckOffTableProps) {
     delay: 1000,
   });
 
+  const attendanceData = attendances.map(attendance => {
+    const fullName = `${attendance.attendee.firstName} ${attendance.attendee.lastName}`;
+    const startTimeString = format(
+      parseISO((attendance.startTime as unknown) as string),
+      'PPP h:mm aaaa'
+    );
+    const attendanceToDisplay = {
+      ...attendance,
+      name: fullName,
+      startTimeString,
+    };
+
+    return attendanceToDisplay;
+  });
+
   // TODO: Remove type casting on startTime when startTime on payload is changed to string and move map logic to a separate function
   return (
-    <>
-      {OfficerRenderPermission(Table)({
-        columns,
-        data: attendances.map(attendance => {
-          const fullName = `${attendance.attendee.firstName} ${attendance.attendee.lastName}`;
-          const startTimeString = format(
-            parseISO((attendance.startTime as unknown) as string),
-            'PPP h:mm aaaa'
-          );
-          const attendanceToDisplay = {
-            ...attendance,
-            name: fullName,
-            startTimeString,
-          };
-
-          return attendanceToDisplay;
-        }),
-        title: 'Check Off Table',
-      })}
-    </>
+    <Table>
+      columns={columns}
+      data={attendanceData}
+      title=Check Off Table
```

Comment:
Doesn't this need to be a string?

Reply:
yeah but eslint won't let me - this looks like it works somehow

Reply:
nvm im troll sorry